bin/check-attachments

```diff
@@ -76,7 +76,7 @@ def main(num_accounts):
 
     with global_session_scope() as db_session:
         accounts = db_session.query(Account).filter(
-            Account.sync_should_run == True).order_by(func.rand()).limit(
+            Account.sync_should_run).order_by(func.rand()).limit(
```

Comment:
Black is obviously not running here

Reply:
Is it valid to filter for `Model.field` without a condition?

Reply:
Even if this is valid, if there's an option of using `.is_(True)` or == true()` (using the true expression provided by sqlalchemy) I'd prefer those.

It's not super clear to me what condition is being evaluated here if there's no obvious operator

Reply:
Yes, `Model.field` in `filter` would just render `WHERE table.field` in SQL which is the same as `WHERE table.field = TRUE` AFAIK.

I agree it can be obscure to a casual reader, and typing a little bit more does not hurt as much as reading.